systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -306,4 +308,22 @@ public void testForUpdateValuesInConfigMap() {
             assertEquals("23", getValueFromJson(zkPodJson, initialDelaySecondsPath));
         }
     }
+
+    @Test
+    @KafkaCluster(name = "my-cluster", kafkaNodes = 3, config = {
+            @CmData(key = "KAFKA_DEFAULT_REPLICATION_FACTOR", value = "1")})
+    @Topic(name = "test-topic", clusterName = "my-cluster")
+    public void testSendMessages() {
+        String topicName = "test-topic";
+        String clusterName = "my-cluster";
+        int messagesCount = 5;
+        List<String> messagesToSend = new ArrayList<>();
+        for (int i = 0; i < messagesCount; i++) {
+            messagesToSend.add("Test message " + i);
+        }
+        sendMessages(messagesToSend, clusterName, topicName);
+        List<String> consumedMessages = consumeMessages(clusterName, topicName);
+        LOGGER.info("Comparing lists of sent and received messages");
+        assertTrue(CollectionUtils.isEqualCollection(messagesToSend, consumedMessages));
```

Comment:
I think that much better to use hamcrest in this case because if the test fails we will have an output like "true is not true". But if you use hamcrest you will have detailed output with elements in List. 
An example:
```
  List<String> actual = Arrays.asList("fee", "fi", "foe");
  List<String> expected = Arrays.asList("fee", "fi", "foe");

  assertThat(actual, is(expected));
  assertThat(actual, is(not(expected)));
```